src/sentry/tasks/integrations/sync_status_inbound.py

```diff
@@ -51,17 +57,78 @@ def sync_status_inbound(
         "provider_key": provider.key,
         "integration_id": integration_id,
     }
-
     if action == ResolveSyncAction.RESOLVE:
+        activity_type = ActivityType.SET_RESOLVED
+        # If the resolution strategy is set to resolve in the next release or current release
+        if config.get("resolution_strategy") in [
+            "resolve_next_release",
+            "resolve_current_release",
+        ]:
+            # only allow setting resolved in release if there is a release for each project
+            all_project_ids = list({group.project_id for group in affected_groups})
+            has_releases_for_each_project = all(
+                Release.objects.filter(
+                    projects=project_id, organization_id=organization_id, status=ReleaseStatus.OPEN
+                ).exists()
+                for project_id in all_project_ids
+            )
+            if has_releases_for_each_project:
+                if config.get("resolution_strategy") == "resolve_next_release":
+                    activity_type = ActivityType.SET_RESOLVED_IN_RELEASE
+                    activity_data["inNextRelease"] = True
```

Comment:
do we need to store the next release version here? https://github.com/getsentry/sentry/blob/777e7de74e140f2bec01a28598bf4fc2cab7c0f0/src/sentry/api/helpers/group_index/update.py#L477